### PR TITLE
2473/cancel claim issue

### LINKED ISSUE #2473: Success screen is displayed when cancel a claim TX
1. Send a Claim TX to the connected wallet, sign it
2. When TX is pending, cancel Claim TX

**AR**: Success screen is displayed after claim TX is cancelled

image

**ER**: Failed Claim screen should be displayed

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes #2473

- fixes the issue with showing success claim view when the transaction is cancelled

# Todo

- [x]  Reset claim repo and address after review

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @nenadV91 , great changes!

One tiny nitpick: when a transaction is cancelled, we get a pop-up notification with a green checkmark on it, so it might seem that the Tx is successful. 
Could we change the icon to the red one (like we use for the failed transactions) for this case? 

Also, just FYI, I have created a separate issue for the case when a transaction is executed, but it hangs in the 'Cancelling' pending state #2486

**Review by W3stside — APPROVED** [review]
did u also change the contract addreses and branch name?

File: src/custom/state/claim/hooks/index.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ import useIsMounted from 'hooks/useIsMounted'
 import { ChainId } from '@uniswap/sdk'
 import { ClaimInfo } from 'state/claim/reducer'
 
-const CLAIMS_REPO_BRANCH = 'gip-13'
```
Comment (anxolin) [inline src/custom/state/claim/hooks/index.ts:65]: Very careful with merging this. If this gets to production we will have some troubles

**Review by anxolin — APPROVED** [review]
Looks ok to me if it passes Elena tests. Pls, don't merge until you revert the branch name

**Review by elena-zh — APPROVED** [review]
@nenadV91 , I have reported a separate issue #2490 for the issue I mentioned above as it is not exactly related to this PR.

**Comment by nenadV91** [comment]
@elena-zh Since there is a separate issue for pop-up notification I am merging this one

**Pull request merged**